tools/packaging/common/envoy_bootstrap.json

```diff
@@ -450,12 +450,14 @@
       {{ end }}
     ],
     "listeners":[
+    {{range $index, $local_ip := .local_ips}}
```

Comment:
Splitting prometheus and status port listeners into 2 each... 1 on localhost and 1 on pod IP. This prevents them from disrupting outbound traffic on ports 15021 and 15090.

Reply:
https://github.com/envoyproxy/envoy/pull/19367 would make this trivial IIUC